typescript/packages/subsurface-viewer/src/layers/wells/utils/generateSyntheticWell.ts

```diff
@@ -17,12 +17,12 @@ export function generateSynteticWell() {
                                 [0, 0, 0],
                                 [0, 0, -100],
                                 [0, 0, -200],
-                                [0, 0, -200],
```

Comment:
You made this earlier to verify well trajectories with duplicate consecutive surveys. Suggest that you create a new example well for the new test. Also, the only thing this function does is return a variable, so might as well just be a variable.